packages/calcite-components/src/components/rating/rating.e2e.ts

```diff
@@ -766,4 +770,59 @@ describe("calcite-rating", () => {
       expect(element).toEqualAttribute("value", "2");
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-rating-spacing": {
+          selector: `calcite-rating`,
+          targetProp: "--calcite-internal-rating-spacing",
```

Comment:
Can you target the CSS property and not the custom CSS prop? We only do the latter for subcomponent mapping.

Reply:
I tried that with this code, but doesn't seem to work.  Am I doing it right?

```
"--calcite-rating-spacing": {
  selector: `.${CSS.fieldset}`,
  targetProp: "gap",
},
```
css
```
.fieldset {
  ...
  gap: var(--calcite-internal-rating-spacing);
}
```

Reply:
Can you test again? The above is passing for me. Here's what my test looks like:



```diff
diff --git i/packages/calcite-components/src/components/rating/rating.e2e.ts w/packages/calcite-components/src/components/rating/rating.e2e.ts
index 927ad059c..0db340f27 100644
--- i/packages/calcite-components/src/components/rating/rating.e2e.ts
+++ w/packages/calcite-components/src/components/rating/rating.e2e.ts
@@ -775,12 +775,10 @@ describe("calcite-rating", () => {
     describe("default", () => {
       const tokens: ComponentTestTokens = {
         "--calcite-rating-spacing": {
-          targetProp: "--calcite-internal-rating-spacing",
+          shadowSelector: `.${CSS.fieldset}`,
+          targetProp: "gap",
         },
         "--calcite-rating-spacing-unit": {
-          targetProp: "--calcite-internal-rating-spacing",
-        },
-        "--calcite-internal-rating-spacing": {
           shadowSelector: `.${CSS.fieldset}`,
           targetProp: "gap",
         },
```

Reply:
Working now, weird.